app/src/test/kotlin/compiler/e2e/CorrectProgramsE2eTest.kt

```diff
@@ -249,4 +250,31 @@ class CorrectProgramsE2eTest {
             compiler.diagnostics.Diagnostic.ParserError::class,
         )
     }
+
+    @Ignore
+    @Test
+    fun `test arrays`() {
```

Comment:
Please add two more tests before this - one per allocation type. It's gonna make it more obvious which allocation type is not implemented correctly during debugging